.eslintrc.js

```diff
@@ -25,6 +25,7 @@ module.exports = {
   'rules': {
     'require-jsdoc': 0,
     'react/prop-types': 0,
+    'prefer-const': 0,
```

Comment:
@danielart I have seen that Preact in their projects always use let, this makes the final bundle smaller. I have tried it and it does, it reduces the size a little bit.